getpaid/backends/transferuj/__init__.py

```diff
@@ -68,19 +70,26 @@ def online(ip, id, tr_id, tr_date, tr_crc, tr_amount, tr_paid, tr_desc, tr_statu
         payment.external_id = tr_id
         payment.description = tr_email
 
-        if tr_status == 'TRUE':
-            # Due to Transferuj documentation, we need to check if amount is correct
-            payment.amount_paid = Decimal(tr_paid)
-            payment.paid_on = datetime.datetime.utcnow().replace(tzinfo=utc)
-            if payment.amount <= Decimal(tr_paid):
-                # Amount is correct or it is overpaid
-                payment.change_status('paid')
-            else:
-                payment.change_status('partially_paid')
-        elif payment.status != 'paid':
+        signals.payment_rejection_available.send(sender=None, payment=payment)
```

Comment:
I have not used Transferuj recently and I’m not able to confirm how it works in practice, however according to the documentation it indeed seems that we should expect `tr_status == 'PAID'` here, but we’re checking for `tr_status == 'TRUE'` (this applies also to the currenct version of Getpaid). Maybe the Transferuj’s behavior has changed…? Look for example at [a Joomla plugin](https://github.com/DodatkiJoomla/transferuj-pl-virtuemart-2/blob/f2223cd8d8f8b0e668e10461c160aa9ccfd30c5d/transferuj.php), there is no string `PAID` there, however there is [a check for `"TRUE"`](https://github.com/DodatkiJoomla/transferuj-pl-virtuemart-2/blob/f2223cd8d8f8b0e668e10461c160aa9ccfd30c5d/transferuj.php#L328).